app/controllers/candidate_interface/invites_controller.rb

```diff
@@ -2,15 +2,53 @@ module CandidateInterface
   class InvitesController < CandidateInterfaceController
     before_action :redirect_to_post_offer_dashboard_if_accepted_deferred_or_recruited
     before_action :redirect_if_feature_off_and_no_submitted_application
+    before_action :set_invite, only: %i[show update course_unavailable]
 
     def index
       @invites = current_application.published_invites.includes(:application_choice)
     end
 
     def show
+      if !@invite.course_open?
+        redirect_to course_unavailable_candidate_interface_invite_path(@invite)
+      end
+
+      @fac_invite_response_form = CandidateInterface::FacInviteResponseForm.new(
+        application_form: current_application,
+        invite: @invite,
+      )
+    end
+
+    def update
+      @fac_invite_response_form ||= CandidateInterface::FacInviteResponseForm.new(fac_invite_response_form_params.merge(
+                                                                                    application_form: current_application,
+                                                                                    invite: @invite,
+                                                                                  ))
+      if @fac_invite_response_form.valid?
+        if @fac_invite_response_form.accepted_invite?
+          @fac_invite_response_form.save
+          redirect_to candidate_interface_course_choices_course_review_path(@invite.application_choice.id)
```

Comment:
We should pass a `return_to` param here so that the user can just click the back link and come back to the edit page.

`retu`
```suggestion
          redirect_to candidate_interface_course_choices_course_review_path(@invite.application_choice.id, return_to: 'edit_invite')
```

We should then use this param in the back link of the draft application choice.